frontend/src/app/teams/components/layout/teams-layout.component.scss

```diff
@@ -1,3 +1,63 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
 .container {
 	height: calc(100vh - 170px);
 }
+
+h1{
+	color: #d8d7d7;
+	font-size: 380px;
+	font-weight: 600;
+	position: absolute;
+	top: 50%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+	z-index: 5;
+	opacity: 0.25;
+}
+
+h2{
+	color: #000;
+	font-size: 30px;
+	margin-bottom: 50px;
+	text-align: center;
+	line-height: 1.4;
+}
+
+.button{
```

Comment:
too specific class names. use class names based on scope like teams__error-button and place it inside .teams__error {
&-...{
}
}